docs/canary-config.md

```diff
@@ -94,6 +95,32 @@ See the [Netflix Automated Canary Analysis Judge] and [Mann Whitney Classifier]
 - `criticalIncrease` **5.0** (number, optional) - Defaults to 1. The multiplier increase that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 5x increase.
 - `criticalDecrease` **0.5** (number, optional) - Defaults to 1. The multiplier decrease that must be met for the metric to be a critical failure and fail the entire analysis with a score of 0. This example make the canary fail critically if there is a 50% decrease.
 
+## Outliers
+
+Controls how to classify and handle outliers.
+Outliers are determined based on the interquartile range (IQR) which is the middle 50% of the data.
+
+The low outlier boundary is calculated using the equation: `Q1 - factor * IQR`.
+The high outlier boundary is calculated using the equation: `Q3 + factor * IQR`. 
+`Q1` and `Q3` are the data values of the lower and upper quartiles respectively.
+`factor` is `outlierFactor` as explained below.
+`IQR` is `Q3 - Q1`.
+
+>Given a latency graph where `Q1` is 50 ms and `Q3` is 100 ms and the `factor` is 3:
+`IQR` is `Q3 - Q1` which is 100 - 50 = 50.
+The low outlier boundary is `Q1 - factor * IQR` which is `50 - 3 * 50 = -50`.
+The high outlier boundary is `Q3 + factor * IQR` which is `100 + 3 * 50 = 250`. So for a data point to be
+classified as an outlier in this example, it must be less than -50 ms or greater than 250 ms.
+
```

Comment:
I think an image would really help clarify this part (perhaps a crop of the top 3rd from https://en.wikipedia.org/wiki/Interquartile_range#/media/File:Boxplot_vs_PDF.svg)